compiler/include/astutil.h

```diff
@@ -75,6 +75,7 @@ void collectDefExprs(BaseAST* ast, llvm::SmallVectorImpl<DefExpr*>& defExprs);
 void collectForallStmts(BaseAST* ast, std::vector<ForallStmt*>& forallStmts);
 void collectCForLoopStmtsPreorder(BaseAST* ast, std::vector<CForLoop*>& cforloopStmts);
 void collectCallExprs(BaseAST* ast, std::vector<CallExpr*>& callExprs);
+void collectCallExprsExceptInGpuBlock(BaseAST* ast, std::vector<CallExpr*>& callExprs);
```

Comment:
I have a slight preference for having this as a `static void` in `gpuTransforms`, but I wouldn't argue strongly for it.

Reply:
This is what I tried at first, but the macros used to implement recursive traversals requires having every AST node included via headers. This seemed not worth the trouble, and I'm inclined to merge as is, particularly since you don't feel strongly about it.

Reply:
I agree with you assessment. Thanks!